huaweicloud/resource_huaweicloud_lb_certificate.go

```diff
@@ -72,6 +75,13 @@ func ResourceCertificateV2() *schema.Resource {
 				DiffSuppressFunc: utils.SuppressNewLineDiffs,
```

Comment:
`certificate` and `private_key` are sensitive arguments, suggest add the attribute with `Sensitive` as true.

Reply:
fixed